src/use-text-as-image.hook.js

```diff
@@ -16,8 +16,10 @@ export function useTextAsImage({processSerializedElement = noop, fontFamily = nu
       richTextContext.selectRangeFromBeforeBlur({usePreviousRange: true})
       const url = textToUrl(text, getSelectedElement(), fontFamily)
       performCommandWithValue(url)
-      const imgElement = document.querySelector(`img[src="${url}"]`)
-      processImgElement(imgElement, text)
+      const imgElement = document.querySelectorAll(`img[src="${url}"]`)
```

Comment:
Instead of selecting all the images and then skipping the ones that have already been processed, I think we could select only the one that hasn't been processed.

```suggestion
      const imgElement = document.querySelector(`img[src="${url}"]:not([data-text-as-image])`)
```

Using the `data-text-as-image` attribute would also prevent us from having to create the extra `data-has-listener` attribute, too.

Reply:
That is a much more efficient solution, thank you!